src/guild/personas.md

```diff
@@ -0,0 +1,10 @@
+# Personas
+
+NQN can allow users to repost messages with custom usernames and avatars. 
+Given the potential for abuse, this feature is disabled by default. 
+
+To help moderation when this feature is enabled, NQN has an audit channel setting, which posts the original author whenever a message is posted via the bot. 
+
+To enable the audit channel, run `/server audit_channel` command, and select the channel you want NQN to post audit message in. 
+
+If you want to see the author of a specific message, you can right click it, go to 'Apps', and then 'Find Author'.
```

Comment:
I feel lines 6 to 10 should be moved to a section detailing moderating messages from NQN all together and be replaced to a link here.